### PR TITLE
Learn Tutorial

### PR DESCRIPTION
**Pull request by clrcrl**
Commit one is a straight copy of the tutorial into a different subdirectory.

Commit two is the changes to make it learn-relevant (I'd just review commit 2).

To navigate to the learn version, you need to go to `/learn/setting-up`

### CODE REVIEW

**Comment by clrcrl** [comment]
Plan to deploy:
1. Maintain a separate branch in this repo
2. Run `npm run build`
3. Create a deployment script on the `learn.getdbt.com` repo that moves the build files into the `_site/` directory
4. Access at learn.getdbt.com/tutorial/getting-started

For now, this is a draft PR so you can comment on the changes

**Comment by clrcrl** [comment]
Check it out [here](https://deploy-preview-15--ecstatic-bartik-ad7eb9.netlify.com/learn/setting-up/)

**Comment by jtcohen6** [comment]
This is looking good. Are we going to share it with students using the Netlify link? I'm fine with that.

We should adjust or remove the "Deploy your project" section. The big unanswered question there is: Should we connect a single "main" project repo to the `dbtlearn` Cloud account, and students just put in their Snowflake credentials? Should each student connect their own repo + create their own environment in the `dbtlearn` Cloud account, so that they can use the IDE to edit their own project?

File: website/docs/learn/5-deploy-your-project.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+## Commit your changes
+First off, we need to commit the changes we made to our project so that our repository has our latest code.
+
+### dbt Cloud
+<LoomVideo id="afd55d89abdc4a77b34deaee90da0813" />
+
+Click the `commit` button, with a message like "Add customers model"
+
+### dbt Core
```
Comment (jtcohen6) [inline website/docs/learn/5-deploy-your-project.md]:
```suggestion
### dbt CLI
```
? I noticed this in the main tutorial as well

File: website/docs/learn/5-deploy-your-project.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+We're very excited to see you at Learn. In the meantime, here's a few suggestions if you want to get a headstart:
+
+### Start working on your own project
+Ready to get started with your own project? We recommend doing the tutorial a second time to create a new project. You should use your own warehouse, along with a query that you frequently run, to build your first models!
+
+### Learn some best practices
+Now that you've got a working dbt project, read more about some of our [best practices](https://docs.getdbt.com/docs/best-practices), or go back this tutorial and read some of the FAQs.
+
+### Keep building this project
```
Comment (jtcohen6) [inline website/docs/learn/5-deploy-your-project.md]: This touches on concepts/extensions we'll cover in dbt Learn, with the expectation that students may or may not be familiar. I think it's great if people are exposed to them sooner, I don't think we need to say anything to that effect here.

File: website/docs/learn/5-deploy-your-project.md (hunk 1 above)
Comment (clrcrl) [reply website/docs/learn/5-deploy-your-project.md]: Thanks!

**Comment by clrcrl** [comment]
@jtcohen6 — can you review this now?

File: website/docs/learn/1-setting-up.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+### FAQs
+<FAQ src="faqs/loading-data" alt_header="The data in this tutorial is already loaded into Snowflake. How do I load data into my warehouse?" />
+<FAQ src="faqs/database-privileges" />
+
+
+## Create a repository
+We're going to use [GitHub](https://github.com/) as our git provider for this tutorial, but you can use any git provider. If you don't yet have a GitHub account, [create one now](https://github.com/join).
+<LoomVideo id="afe148aeab5e4279a2ca310251ea20a6" />
+
+1. Create a new GitHub repository [here](https://github.com/new) named `dbt-learn-[initiallastname]` (e.g. `dbt-learn-ccarroll`).
```
Comment (jtcohen6) [inline website/docs/learn/1-setting-up.md]:
```suggestion
1. Create a new GitHub repository [here](https://github.com/new) named `dbt-learn-[initialsurname]` (e.g. `dbt-learn-ccarroll`).
```
Later on you call this `initialsurname`, which I prefer, I think it's easier to read that way

**Pull request merged**